.github/workflows/verdi.sh

```diff
@@ -37,3 +37,19 @@ done
 
 $VERDI devel check-load-time
 $VERDI devel check-undesired-imports
+
+
+# Test that we can also run the CLI via `python -m aiida`,
+# that it returns a 0 exit code, and contains the expected stdout.
+echo "Invoking verdi via `python -m aiida`"
+OUTPUT=$(python -m aiida 2>&1)
+RETVAL=$?
+echo $OUTPUT
+if [ $RETVAT -ne 0 ]; then
```

Comment:
Uhm @chrisjsewell this may have been my mistake in the review suggestion, but there is a type here. If I promise to not merge your PRs anymore, will you promise to let me sign off on them first :)

Reply:
Don't worry, I have a small PR coming up with some minor corrections that I can put this in as well

Reply:
cheers!